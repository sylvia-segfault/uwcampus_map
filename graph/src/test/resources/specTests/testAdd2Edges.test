# Create an empty graph
CreateGraph graph3

# Add 3 nodes to graph3
AddNode graph3 node1
AddNode graph3 node2
AddNode graph3 node3

# Add 2 edges to graph3
AddEdge graph3 node1 node2 edge1
AddEdge graph3 node1 node3 edge2