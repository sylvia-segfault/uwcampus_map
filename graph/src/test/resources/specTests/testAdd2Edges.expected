# Create an empty graph
created graph graph3

# Add 3 nodes to graph3
added node node1 to graph3
added node node2 to graph3
added node node3 to graph3

# Add 2 edges to graph3
added edge edge1 from node1 to node2 in graph3
added edge edge2 from node1 to node3 in graph3